sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveExplainSuite.scala

```diff
@@ -74,4 +74,21 @@ class HiveExplainSuite extends QueryTest {
       "Limit",
       "src")
   }
+
+  test("SPARK-6212: The EXPLAIN output of CTAS only shows the analyzed plan") {
+    checkExistence(sql(
+      s"""
+         |EXPLAIN EXTENDED
+         |CREATE TABLE t1
+         |AS
+         |SELECT * FROM src
+      """.stripMargin), true,
+      "== Parsed Logical Plan ==",
+      "== Analyzed Logical Plan ==",
+      "== Optimized Logical Plan ==",
+      "== Physical Plan ==",
+      "CreateTableAsSelect",
+      "InsertIntoHiveTable",
+      "src")
```

Comment:
I feel this one is pretty much the same as the test in `explain create table command` (the above test). I think we need to have a test to make sure that the physical plan output string does not contain `Subquery`.